doc/release/1.14.0-notes.rst

```diff
@@ -558,6 +558,29 @@ Reduced memory usage of ``np.loadtxt``
 --------------------------------------
 ``np.loadtxt`` now reads files in chunks instead of all at once which decreases
 its memory usage significantly for large files.
+`numpy.errstate` is now also function decorator in Python 3
+-----------------------------------------------------------
+`numpy.errstate` is now also function decorator
+-----------------------------------------------
```

Comment:
Merge conflict here. Needs to move to the 1.15 release notes anyway - sorry for not being faster on this